ansible_base/activitystream/serializers.py

```diff
@@ -0,0 +1,8 @@
+from ansible_base.activitystream.models import Entry
+from ansible_base.lib.serializers.common import ImmutableCommonModelSerializer
+
+
+class EntrySerializer(ImmutableCommonModelSerializer):
+    class Meta:
+        model = Entry
+        fields = ImmutableCommonModelSerializer.Meta.fields + ['operation', 'changes']
```

Comment:
Isn't this missing a lot of fields? What about including `content_object` in the summary_fields (I did that in RBAC). Also for you, `related_content_object`.